docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Forgot your password or username
+
+Via [{{gui}}](https://{{gui_domain}}) you can reset lost password or reclaim your username.
+
+## Reset lost password
+
+Navigate to <https://{{gui_domain}}> and click _Forgot your password?_.
+
+![Picture with a orange button with white text Forgot your password? framed with red color](assets/Forgot-your-password.png)
+
+This image appears. Enter your username and click _Reclaim password!_.
+
+![Picture with a field Enter your username and a orange button with white text Reclaim password! framed with red color](assets/Reclaim-lost-password.png)
+
+Now you will see a page with this information: _An email has been sent to you with instructions on how to reset your password_. 
+
+Click Ok! and check your email inbox with the subject **_Cleura Account password reset request_**.
+
+![Picture with text information An email has been sent to you with instructions on how to reset your password and a orange button with white text Ok! framed with red color](assets/OK-reclaim-lost-password.png)
+
+The content of the email will look like the image below. Click on the _link_ to create a new password.
+
+![Picture with information to follow the link and create a new password](assets/Reset-password-link.png)
+
+After clicking on the link you will be redirected to a new page. Fill in both fields with new password and click _Set new password_.
+
+![Picture how you fill in your new password and a orange button with white text Set new password framed with red color](assets/Choose-new-password.png)
+
+**Now you completed reset your lost password**.
+
+![Picture with information that your new password was succesfully changed](assets/Reset-completed.png)
+
+## Reclaim your username
+
+Navigate to <https://{{gui_domain}}> and click _Forgot your password?_.
```

Comment:
Suggestion: "Navigate to the <https://{{gui_domain}}>, and at the bottom left-hand side of the page click the _Forgot your password?_ button."